_layouts/post.html

```diff
@@ -8,19 +8,36 @@ <h1 class="post-title">{{ page.title }}</h1>
   {{ content }}
 </div>
 
-{% if site.related_posts.size >= 1 %}
+{% assign full_name = page.chapter | prepend: "chapter" %}
+{% assign post_list = site.categories[full_name] | sort_natural: "order" %}
+
+{% assign post_order = page.order%}
+
+{% assign post_before_before = page.order | minus: 2 %}
+{% assign post_before = page.order | minus: 1 %}
+{% assign post_after = page.order | plus: 1 %}
+{% assign post_after_after = page.order | plus: 2 %}
+
+{% if post_list.size >= 2 %}
 <div class="related">
   <h2>Related posts</h2>
   <ul class="related-posts">
-    {% for post in site.related_posts limit:3 %}
-      <li>
-        <h3>
-          <a href="{{ site.baseurl }}{{ post.url }}">
-            {{ post.title }}
-            <small>{{ post.date | date_to_string }}</small>
-          </a>
-        </h3>
-      </li>
+    {% for post in post_list%}
+      {% if 
+        post.order == post_before_before or 
+        post.order == post_before or 
+        post.order == post_after or
+        post.order == post_after_after 
+      %}
```

Comment:
```suggestion
    {% for post in post_list%}
      {% if 
        post.order == post_before or 
        post.order == post_after or
      %}
```
직전과 직후만 있어도 될 것 같습니다

Reply:
@RRoundTable  @hunhoon21 @LEEMINJOO   ㅋㅋㅋㅋㅋ 그냥 저렇게 하나씩 assign 하는 방식이 마음에 안들었습니다. 찾아봤는데 liquid 언어 특성상 딱히 좋은 방법이 떠오르지 않더라고요ㅠㅋㅋ

Reply:
수정 완료했습니다!